src/plugins/navigation/public/top_nav_menu/create_top_nav_menu.tsx

```diff
@@ -24,9 +23,9 @@ export function createTopNav(
     const config = (props.config || []).concat(relevantConfig);
 
     return (
-      <i18n.Context>
+      <I18nProvider>
```

Comment:
ℹ️ I am changing this because it creates a conflict with EUI and the comma here
![image](https://user-images.githubusercontent.com/17003240/163413380-4fa7e7d3-301b-4b36-afae-d5fa9e00224e.png)
was translated as [Object object]